canvas_modules/common-canvas/locales/common-properties/locales/en.json

```diff
@@ -36,11 +36,11 @@
   "alerts.tab.title": "Alerts",
   "title.editor.label": "edit title",
   "title.editor.helpButton.label": "help",
-  "table.summary.error": "There are {errorMsgCount} error cells. ",
-  "table.summary.warning": "There are {warningMsgCount} warning cells. ",
-  "control.summary.error": "There are {errorMsgCount} parameters with errors. ",
-  "control.summary.warning": "There are {warningMsgCount} parameters with warnings. ",
-  "control.summary.error.warning": "There are {errorMsgCount} parameters with errors and {warningMsgCount} parameters with warnings.",
+  "table.summary.error": "There are {errorMsgCount, plural, one {# error cell} other {# error cells} }. ",
+  "table.summary.warning": "There are {warningMsgCount, plural, one {# warning cell} other {# warning cells} }. ",
+  "control.summary.error": "There are {errorMsgCount, plural, one {# parameter with error} other {# parameters with errors} }. ",
+  "control.summary.warning": "There are {warningMsgCount, plural, one {# parameter with warning} other {# parameters with warnings} }. ",
+  "control.summary.error.warning": "There are {errorMsgCount, plural, one {# parameter with error} other {# parameters with errors} } and {warningMsgCount, plural, one {# parameter with warning} other {# parameters with warnings} }.",
```

Comment:
These messages don't really work because, for a single value, they would say something like:
 "There are 1 error cell." 
 whereas with singular it should be
 "There is 1 error cell."
 and for plural:
 "There are 3 error cells"